test.js

```diff
@@ -1,8 +1,14 @@
 'use strict';
 
 /* eslint-disable no-process-exit */
-if(!process.env.CLOUDFLARE_EMAIL || !process.env.CLOUDFLARE_APIKEY){
-	console.error('Missing CLOUDFLARE_EMAIL or CLOUDFLARE_APIKEY env');
+if(!process.env.CLOUDFLARE_TOKEN && !(process.env.CLOUDFLARE_EMAIL && process.env.CLOUDFLARE_APIKEY)){
+	console.error(`Missing CLOUDFLARE_TOKEN or ${
+		process.env.CLOUDFLARE_EMAIL ? '' : 'CLOUDFLARE_EMAIL'
+	}${
+		process.env.CLOUDFLARE_EMAIL && process.env.CLOUDFLARE_APIKEY ? ' and ' : ''
+	}${
+		process.env.CLOUDFLARE_APIKEY ? '' : 'CLOUDFLARE_APIKEY'
+	} env`);
```

Comment:
A multi-line log like this with multiple ternary operators is pretty unreadable. Can we perhaps move this to separate logs, or create a variable to determine what's set/missing?